### ISSUE TITLE
Thin corpus callosum

### ISSUE THREAD

**Issue by DLewisSmith** [obophenotype/human-phenotype-ontology#6731]
**Preferred term label:**
Thin corpus callosum

**Synonyms**
Small corpus callosum

**Definition (free text, please give PubMed ID)**
An abnormally thin corpus callous, due to atrophy, hypoplasia or agenesis.

This is useful for the common situation where MRI is reported as showing 'thinning of the corpus callosum' but it is not known if this is due to abnormal development (e.g. a leukodystrophy) or atrophy following normal development (e.g. neurodegeneration).

**Parent term (use hpo.jax.org/app)**
Proposed term has parent:
Abnormal corpus callosum morphology HP:0001273

Proposed term has children:
Aplasia/Hypoplasia of the corpus callosum HP:0007370
Corpus callosum atrophy HP:0007371

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Those associated with its children, leukodystorphies, neurodegenerative disorders, and most commonly advanced multiple sclerosis.

**Your nano-attribution (ORCID)**
0000-0002-1735-8178

**Comment by pnrobinson** [comment]
good suggestion, thanks!
